contracts/periphery/Periphery.sol

```diff
@@ -244,7 +262,7 @@ contract Periphery is IPeriphery {
             vamm.initializeVAMM(sqrtRatioAtMidTickX96);
         }
 
-        if (params.marginDelta != 0) {
+        if (params.marginDelta != 0 || msg.value > 0) {
```

Comment:
Again, would it be simpler and more intuitive to have marginDelta include the value of any ETH?